configure.ac

```diff
@@ -5871,7 +5871,12 @@ AC_ARG_WITH(ensurepip,
     [AS_HELP_STRING([--with-ensurepip@<:@=install|upgrade|no@:>@],
         ["install" or "upgrade" using bundled pip (default is upgrade)])],
     [],
-    [with_ensurepip=upgrade])
+    [
+      AS_CASE([$ac_sys_system],
+        [Emscripten], [$with_ensurepip=no],
```

Comment:
Should we disable ensurepip for WASI as well?

Reply:
I prefer to wait until we have WASI builds working and more experience with WASI.